### ISSUE TITLE
Enable auto merge after 24 hours without any comments

### ISSUE THREAD

**Issue by tkoyama010** [pyvista/pyvista#2774]
### Describe what maintenance you would like added.

I would like to refrain from merging for at least 24 hours after I submit my LGTM. During that time the earth will turn into a full revolution and all interested reviewers will be given the opportunity to review. However, I would prefer to automatically merge after 24 hours if there are no comments. I will have to look at the same PR again after 24 hours without comments. The current enable auto marge does not have the option to specify a time, I wish GitHub Action could do that.

_Originally posted by @tkoyama010 in https://github.com/pyvista/pyvista/discussions/2245#discussioncomment-2838295_

### Links to source code.

None

### Pseudocode or Screenshots

One way to implement that would be to create a github action cronjob that runs nightly and checks if there are any PRs that have been approved without additional comments after the initial approval. I'm thinking of using [pygithub](https://github.com/PyGithub/PyGithub). We can add it to the "to-do" list unless someone has a better approach

_Originally posted by @akaszynski in https://github.com/pyvista/pyvista/discussions/2245#discussioncomment-2838334_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MatthewFlamm** [comment]
I would prefer if this is opt-in.  For example, auto merge if the approval comment contains "merge next day" or something like that.  There will be situations where I may approve, but will want others to also approve or at least consider waiting for another set of eyes.

**Comment by tkoyama010** [comment]
Thanks for your comment. The difficulty with this feature is that it is not simply a matter of merging after 24 hours; we have to make sure that no comments have been entered by other users.

**Comment by MatthewFlamm** [comment]
I agree, I'm asking for more complexity unfortunately.  If the approver opts-in, then merge if no additional comments are made after X hrs.
